src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -281,19 +285,19 @@ protected WebResourceResponse interceptRequest(String rawUrl) {
       res = new WebResourceResponse("text/plain", "UTF-8", targetStream);
     }
 
-    if (res != null) {
-      Map<String, String> headers = new HashMap<>();
-      headers.put("Content-Security-Policy",
-          "default-src 'self'; "
-        + "style-src 'self' 'unsafe-inline' blob: ; "
-        + "font-src 'self' data: blob: ; "
-        + "script-src 'self' 'unsafe-inline' 'unsafe-eval' blob: ; "
-        + "connect-src 'self' data: blob: ; "
-        + "img-src 'self' data: blob: ; "
-        + "media-src 'self' data: blob: ;"
-        + "webrtc 'block' ; "
-      );
-      headers.put("X-DNS-Prefetch-Control", "off");
+    Map<String, String> headers = new HashMap<>();
+    headers.put("Content-Security-Policy",
+        "default-src 'self'; "
+      + "style-src 'self' 'unsafe-inline' blob: ; "
+      + "font-src 'self' data: blob: ; "
+      + "script-src 'self' 'unsafe-inline' 'unsafe-eval' blob: ; "
+      + "connect-src 'self' data: blob: ; "
+      + "img-src 'self' data: blob: ; "
+      + "media-src 'self' data: blob: ;"
+      + "webrtc 'block' ; "
+    );
+    headers.put("X-DNS-Prefetch-Control", "off");
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
we're not opening the whole activity if we're not running on `>= Build.VERSION_CODES.LOLLIPOP` , however, to remove a warning, the change makes sense.

consider adding also the header construction inside the if:

```suggestion
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
      Map<String, String> headers = new HashMap<>();
      headers.put("Content-Security-Policy",
          "default-src 'self'; "
        + "style-src 'self' 'unsafe-inline' blob: ; "
        + "font-src 'self' data: blob: ; "
        + "script-src 'self' 'unsafe-inline' 'unsafe-eval' blob: ; "
        + "connect-src 'self' data: blob: ; "
        + "img-src 'self' data: blob: ; "
        + "media-src 'self' data: blob: ;"
        + "webrtc 'block' ; "
      );
      headers.put("X-DNS-Prefetch-Control", "off");
```